docs/rest_api/getting_started.rst

```diff
@@ -0,0 +1,99 @@
+Getting started
+###############
+
+Mautic provides several ``REST API`` endpoints that you can use. In the navigation menu on the left, you can find the available endpoints.
+
+.. warning:: 
+
+    By default, Mautic's ``REST API`` isn't enabled. The Mautic administrator can enable the API in the Mautic UI under ``Configuration -> API Settings``, or by setting ``'api_enabled' => 1`` in ``app/config/local.php`` directly.
+
+.. note:: 
+
+    Mautic has an API library available for PHP. You can `find it on GitHub <https://github.com/mautic/api-library>`_.
+
+To get started easily, you can use Mautic's Postman collection:
+
+.. image:: https://run.pstmn.io/button.svg
```

Comment:
This is very incomplete right now, I'd personally leave this out until we fill it out a bit.